src/components/Modal/Modal.tsx

```diff
@@ -39,6 +39,8 @@ const CloseButtonContainer = styled(Button.Container)`
   ${({ closeButtonAttachment }: { closeButtonAttachment: string }) => {
     let distance;
     let position;
+    let display_type = 'flex'; //what is the desired default for the close button's display-type??
```

Comment:
default distance and position to something here
update display_type to `inline-flex`

change `display_type` to `display`